appintro/src/main/java/com/github/appintro/AppIntroBase.kt

```diff
@@ -382,6 +388,7 @@ abstract class AppIntroBase : AppCompatActivity(), AppIntroViewPagerListener {
      LIFECYCLE
      =================================== */
 
+    @Suppress("DEPRECATION")
```

Comment:
why do we need this if we removed the `setScrollDurationFactor` call?

Reply:
I just removed the method from the example, that's a call in the internal `AppIntroBase` activity in the library.

Actually this sets `setScrollDurationFactor(DEFAULT_SCROLL_DURATION_FACTOR)` during intro's `onCreate`. I'm not entirely sure why we need that, since the duration factor is already initialized to `DEFAULT_SCROLL_DURATION_FACTOR`.

But anyway, it was there in the first place and I'm scared to introduce a regression in a bugfix release because probably there's a side effect I'm not considering.

Reply:
Ah sorry I thought it's in the sample code, my bad. It's fine to stay here.